crates/cli/tests/tests.rs

```diff
@@ -202,6 +210,7 @@ fn benchmark_effect_size() -> anyhow::Result<()> {
     let alt_engine_path: PathBuf = alt_engine.path().into();
     alt_engine.close()?;
     std::fs::copy(&test_engine, &alt_engine_path)?;
+    assert!(alt_engine_path.exists());
```

Comment:
@fitzgen, any ideas? I'm going to merge this tomorrow unless you object because it seems to fail on Windows only and, when I re-run the jobs, it then seems to work. Not really sure if this added line does much...

Reply:
It seems like kind of a silly assert since we just did the copy on the line right above.